src/components/index.d.ts

```diff
@@ -747,10 +747,14 @@ export declare const EmailInput: {
     labelClasses?: string
     validator?: (value: string) => string
     initialValue?: string
+    iconPrefix?: string
+    iconName?: string
```

Comment:
If there are only 3 icons being made available, it seems like this could be a single `icon` parameter to me, with a type that checks the value, if present, is equal to one of [bell, lock, eyeSlash]. I think if we want to limit the user to only these 3 icons, we should be more explicit here so there is a clear source of truth for what icons are valid when type checking, without having to read into the conditional statements in the individual components.

Reply:
Resolving since we discussed on zoom.

Reply:
Updated as discussed. Thank you Cole @coleEthos @ChirashiSushi